server/api.js

```diff
@@ -240,10 +240,10 @@ router.put("/v1/processes/:ref/transfer", (req, res) => {
     matApiDataIV
   );
 
-  const assignedToManager = processStage === "1";
+  const assignedToManager = !(processStage === "1");
```

Comment:
```suggestion
  const transferToOfficer = processStage === "1";
```

If I understand correctly, this is more descriptive. It's saying what _should_ happen, rather than what the current state is (which is what the existing name implies). This change would mean reversing the other change in this PR and fixing the options to be:

```
      assignedToPatch: transferToOfficer,
      assignedToManager: !transferToOfficer,
```

Reply:
We've changed the what should happen scenario to transfer to Manager instead of transfer to Officer as this follows the logic needed.